share/antioch_default_files/antioch_default_chemical_mixture.dat

```diff
@@ -80,6 +80,7 @@ O	16.00000	  1.5420000000e7 	1.5 	 0
 O+	15.99945	  9.7560000000e7 	1.5 	 1
 O2	32.00000	  0.000000000000 	2.5 	 0
 O2+	31.99945	  3.6370000000e7 	2.5 	 1
+O3      48.00000          3.0104166666e6        3.0      0
```

Comment:
Is this also from http://cccbdb.nist.gov/hf0K.asp ?

Reply:
Correct. I should've included in the commit log that the comment exactly as stated is still correct. I'll force push an amendment to the commit log.